app/utility/transforms/Validator.js

```diff
@@ -0,0 +1,9 @@
+import OpenBarbellConfig from 'app/configs/OpenBarbellConfig.json';
+
+export const validator = () => {
+    if (this.lastRefreshDate < OpenBarbellConfig.obtainTokenTimer) {
```

Comment:
I'm iffy about the naming with "obtain" as a config file isn't really a selector / getter, it just contains the properties themselves.